src/Analyzers/Microsoft.Analyzers.Local/ApiLifecycle/Utils.cs

```diff
@@ -90,6 +90,6 @@ private static void GetBaseTypesImpl(List<string> results, string baseTypesStrin
             }
         }
 
-        results.Add(baseTypesString.Substring(start, baseTypesString.Length - start).Trim());
+        results.Add(baseTypesString.Substring(start).Trim());
```

Comment:
This and a couple of the other line changes in the PR are unrelated and are just addressing some easy analyzer suggestions in the solution.